tests/test_backend.py

```diff
@@ -1787,6 +1787,32 @@ def func(x, y):
             return tf.identity(x_, name=_TFOUTPUT)
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: y_val})
 
+    @check_opset_min_version(10, "Slice")
+    def test_strided_slice_reverse_1(self):
+        tf.reset_default_graph()
```

Comment:
use tf_reset_default_graph() because of tf-2.x. 
But it is not needed because we do it implicit before calling func.

Reply:
Thanks -- removed.